packages/date-picker/src/vaadin-month-calendar.js

```diff
@@ -110,8 +110,11 @@ class MonthCalendar extends MonthCalendarMixin(ThemableMixin(PolymerElement)) {
   // eslint-disable-next-line @typescript-eslint/max-params
   __getDatePart(date, focusedDate, selectedDate, minDate, maxDate, isDateDisabled) {
     const result = ['date'];
+    const dayDisabled = this.__isDayDisabled(date, minDate, maxDate, isDateDisabled);
+    const greaterThanToday = normalizeDate(date) > normalizeDate(new Date());
+    const lessThanToday = normalizeDate(date) < normalizeDate(new Date());
```

Comment:
I checked, and the incoming date has no time part already due to the [way it's created](https://github.com/vaadin/web-components/blob/adf4da0bfb0bbb538932a9dee6c2f350ac368b7e/packages/date-picker/src/vaadin-month-calendar-mixin.js#L243).

```suggestion
    const greaterThanToday = date > normalizeDate(new Date());
    const lessThanToday = date < normalizeDate(new Date());
```